apps/arena/lib/arena/game_updater.ex

```diff
@@ -1230,6 +1246,45 @@ defmodule Arena.GameUpdater do
     end)
   end
 
+  defp update_bounties_states(%{status: :RUNNING} = game_state, state) do
+    actual_players =
+      state.clients
+      |> Enum.map(fn client_id ->
+        player_id =
+          get_in(game_state, [:client_to_player_map, client_id])
+
+        player = get_in(game_state, [:players, player_id])
+
+        {player_id, player}
+      end)
```

Comment:
I don't get why you need, isn't this exactly the same as `Map.to_list(game_state.players)`? And if its only to use it in `Enum.reduce/3` then you should be able to just past and it will give you the same

Reply:
nope, it's not the same as doing game_state.players since here we only got the real players and not bot players, since we don't care about the bots getting currencies. It's something similar to what game_tracker is doing since the `clients` field from the `game_state` only has humans. 
I'll rename `actual_players` to `human_players` to avoid confusion